packages/liquidsoap/liquidsoap.2.0.6/opam

```diff
@@ -0,0 +1,187 @@
+opam-version: "2.0"
+synopsis: "Swiss-army knife for multimedia streaming"
+description: """\
+Liquidsoap is a powerful and flexible language for describing your
+streams. It offers a rich collection of operators that you can combine
+at will, giving you more power than you need for creating or
+transforming streams. But liquidsoap is still very light and easy to
+use, in the Unix tradition of simple strong components working
+together."""
+maintainer: "romain.beauxis@gmail.com"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+license: "GPL-2.0-or-later"
+homepage: "https://github.com/savonet/liquidsoap"
+bug-reports: "https://github.com/savonet/liquidsoap/issues"
+depends: [
+  "conf-autoconf" {dev}
+  "conf-automake" {dev}
+  "ocaml" {>= "4.12.0" & < "5.0.0"}
+  "camomile" {>= "1.0.0"}
+  "dtools" {>= "0.4.4"}
+  "duppy" {>= "0.9.1"}
+  "menhir" {>= "20180703" & <= "20211128"}
```

Comment:
Sorry for the delay, we seem to have forgotten about this PR. One question before I merge, is it still required?
```suggestion
  "menhir" {>= "20180703"}
```

Reply:
I think so, yes.

Reply:
I just tried with 20220210 and it compiled just fine locally